feature-styles/US_EnvironmentalManagementFacility.se

```diff
@@ -1,52 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<se:FeatureTypeStyle xmlns:us-emf="http://inspire.ec.europa.eu/schemas/us-emf/4.0" 
+<se:FeatureTypeStyle xmlns:us-emf="http://inspire.ec.europa.eu/schemas/us-emf/4.0"
 xmlns:act-core="http://inspire.ec.europa.eu/schemas/act-core/4.0"
-xmlns:ogc="http://www.opengis.net/ogc" xmlns:se="http://www.opengis.net/se" 
-xmlns:sld="http://www.opengis.net/sld" xmlns:xlink="http://www.w3.org/1999/xlink" 
+xmlns:ogc="http://www.opengis.net/ogc" xmlns:se="http://www.opengis.net/se"
+xmlns:sld="http://www.opengis.net/sld" xmlns:xlink="http://www.w3.org/1999/xlink"
 xsi:schemaLocation="http://www.opengis.net/se http://schemas.opengis.net/se/1.1.0/FeatureStyle.xsd"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1.0">
-	<se:Name>US.EnvironmentalManagementSite.Default</se:Name>
+	<se:Name>US.EnvironmentalManagementFacility</se:Name>
 	<se:Description>
-    	<se:Title>Environmental Management Facilities</se:Title>
+    	<se:Title>Environmental Management Facility</se:Title>
     	<se:Abstract>Point geometries are rendered as a triangle with a size of 5 pixels, with a 50% grey (#808080) fill and a black outline. Line geometries are rendered as a solid black line with a stroke width of 1 pixel.
 		Polygon geometries are rendered using a 50% grey (#808080) fill and a solid black outline with a stroke width of 1 pixel</se:Abstract>
 	</se:Description>
-	<se:FeatureTypeName>us-emf:EnvironmentalManagementFacility</se:FeatureTypeName> 
-	<se:Rule>
-    <se:Description>
-      <se:Title>Environmental management facility default</se:Title>
-    </se:Description>
-    <se:PolygonSymbolizer>
-      <se:Geometry>
-	        <ogc:PropertyName>act-core:geometry</ogc:PropertyName>
-	   </se:Geometry>
-	   <se:Fill>
-	     <se:SvgParameter name="fill">#808080</se:SvgParameter>
-	   </se:Fill>
-	   <se:Stroke>
-	      <se:SvgParameter name="stroke">#000000</se:SvgParameter>
-	      <se:SvgParameter name="stroke-width">1</se:SvgParameter>
-	   </se:Stroke>
-    </se:PolygonSymbolizer>
-  </se:Rule>
+	<se:FeatureTypeName>us-emf:EnvironmentalManagementFacility</se:FeatureTypeName>
   	<se:Rule>
-  	 <se:Description>
-      <se:Title>Site</se:Title>
-    </se:Description>
+			<se:Description>
+	      <se:Title>Environmental management facility default</se:Title>
```

Comment:
Not sure where this value is coming from, cannot find it in the DS. But not sure if all values would occur there at all.

Reply:
This is the value that is displayed in the legend. The user can determine this. It is not tested.